aether-common-module/aether/common/conf/tests/test_settings.py

```diff
@@ -32,3 +33,7 @@ def test_default_variables(self):
         self.assertEqual(settings.SECURE_PROXY_SSL_HEADER, None)
 
         self.assertEqual(settings.ROOT_URLCONF, 'aether.common.urls')
+
+    def test_custom_storage_variables(self):
+        with self.settings(DEFAULT_FILE_STORAGE='storages.backends.s3boto3.S3Boto3Storage'):
+            self.assertEqual(str(default_storage.__class__), "<class 'storages.backends.s3boto3.S3Boto3Storage'>")
```

Comment:
This is not really testing anything but django internal storage machinery. There doesn't seem there's anything to be unit-tested in this PR.

Reply:
So, are we removing these tests?